Add KVM setup to quickstart.md (#689)
flatcar/flatcar-website: content/docs/latest/getting-started/quickstart.md

cp -i --reflink=auto flatcar_production_qemu_image.img.fresh flatcar_production_qemu_image.img
```

## Enable hardware-accelerated virtualization

> On Linux, QEMU uses KVM for hardware-accelerated virtualization. Your local user needs to be a member of the `kvm` group. You only need to do this once.

Reviewer: well i'd have rather simply commited the last suggestion by copilot, seems like you rewrote it. "On Ubuntu and in WSL2 on Windows, QEMU can use KVM for hardware-accelerated virtualization. Your local user needs to be a member of the `kvm` group. You only need to do this once." lets keep this to match the scope of the pr to ubuntu and windows. allthough i see you hadd similar written on the last commit sorry for vouching for it change, didn't pay much attention. i think its also safe to revert back to old.

> On Linux, QEMU uses KVM for hardware-accelerated virtualization. Your local user needs to be a member of the `kvm` group. You only need to do this once.

Author: I have been highly confused about the wording, ngl. The original commit was following the learning series verbatim, so that both the learning series and quickstart align. But the copilot kept confusing me, hence the current "_On Ubuntu and in WSL2 on Windows, QEMU can use `KVM` for hardware-accelerated virtualization. Your local user needs to be a member of the `kvm` group. You only need to do this once._"  So I thought maybe I could use the maintainer's help, hence didn't change the PR description.  
Should I tag a maintainer for help in this regard?

> On Linux, QEMU uses KVM for hardware-accelerated virtualization. Your local user needs to be a member of the `kvm` group. You only need to do this once.

Reviewer 2: ```suggestion
On Linux-based systems, QEMU uses KVM for hardware-accelerated virtualization. The current user needs to be a member of the `kvm` group.
```

Check whether your user is already in the `kvm` group:

```bash

> id

Reviewer 2: ```suggestion
id -nG
```

```

> If `kvm` is not listed in the groups section, add your user:

Reviewer 2: ```suggestion
If `kvm` is not listed, Please proceed with adding your user to the `kvm` group. Note that adding the user to a group is an one-time configuration step. The group membership persists across subsequent login sessions and system reboots.
```

> If `kvm` is not listed, Please proceed with adding your user to the `kvm` group. Note that adding the user to a group is an one-time configuration step. The group membership persists across subsequent login sessions and system reboots.

Reviewer 2: Can you please apply this suggestion?
